modules/ROOT/pages/scatter-gather-concept.adoc

```diff
@@ -91,52 +96,88 @@ Example of handling these errors:
+If you configure a timeout for a Scatter-Gather component and a route does not complete processing before the timeout expires, the route throws a `MULE:TIMEOUT` error. This error is then handled the same way as any other error generated from a route: after each route completes (either by processing successfully or by throwing a `MULE:TIMEOUT` error), the successful results and errors are collected into the Scatter-Gather component `MULE:COMPOSITE_ROUTING` error, which is then processed in your configured error handler.
 
 == Example Project
-In Anypoint Studio, you can download and open the example project _Scatter-Gather Flow Control_ from Anypoint Exchange to learn more about how to use the Scatter-Gather component. This example shows the usage of the scatter-gather control flow to aggregate data in parallel and return the result in JSON.
+In Anypoint Studio, you can download and open the example project https://www.mulesoft.com/exchange/org.mule.examples/scatter-gather-flow-control/[Scatter-Gather Flow Control^] from Anypoint Exchange to learn more about how to use the Scatter-Gather component. This example shows the use of the scatter-gather control flow to aggregate data in parallel and return the result in JSON.
 
-The example uses prepared data as input for two resources that should be aggregated. The data represents information about two contacts and has the following structure:
+The example uses prepared data as input for two resources to be aggregated. The data represents information about two contacts and has the following structure:
 
 |===
 |Resource|`firstname`|`surname`|`phone`|`email`
 |contacts-1.csv
 |John
 |Doe
 |096548763
-|john.doe@texasComp.com
+|+john.doe@texasComp.com+
 
 |contacts-2.csv
 |Jane
 |Doe
 |091558780
-|jane.doe@texasComp.com
+|+jane.doe@texasComp.com+
 |===
 
-DataWeave is used to aggregate the data. The information about the contacts is aggregated to a JSON structure that represents data from both resources.
+The example uses DataWeave to aggregate the data. The contact information is aggregated to a JSON structure that represents data from both resources.
+
+To download and open this example project through Anypoint Studio, click the Exchange icon in the upper-left corner. Then, in the window that opens, log into Anypoint Exchange and search on the name of the project.
 
-To download and open this example project while you are in Anypoint Studio, click the Exchange icon in the upper-left corner. Then, in the window that opens, log into Anypoint Exchange and search on the name of the project.
+== Repeatable Stream Input
+
+The Scatter-Gather component can process only repeatable streams. Scatter-Gather cannot process a non-repeatable stream as input from an upstream component.
```

Comment:
```suggestion
The Scatter-Gather component can process only repeatable streams. Scatter-Gather cannot process a non-repeatable stream, which it can receive as input from an upstream component that is configured with a non-repeatable streaming strategy.
```